### ISSUE TITLE
https://www.liens-telechargement.com (redirection)

### ISSUE THREAD

**Issue by GLoBoLLo** [reek/anti-adblock-killer#3743]
Hello,
Blocker is detected on all links on this site, for exemple :

https://www.liens-telechargement.com/123455601123455603123455610123455615123455617wg9zwk9wca

It come from a direct download site :  https://ww1.zone-telechargement.ws
All links for downloading are like this one and detect uBlock Origin.

**Comment by uBlock-user** [comment]
`@@||liens-telechargement.com^$elemhide`

@jspenguin2017

**Comment by jspenguin2017** [comment]
Can't reproduce.

**Comment by GLoBoLLo** [comment]
Thank you for your quick reply. 
I added
@@ || links-shared.com ^ $ elemhide
in "My Filters" from uBlock Origin, but uBlock is still detected :(

**Comment by uBlock-user** [comment]
clear the cache and site data, works for me.

**Comment by GLoBoLLo** [comment]
Thanks.
I pasted it exactly as you posted (it's the translator that added spaces, I use it to check my basic English, and I just saw that it did a bad translation of the website address ;) )
I cleared the cache, deleted cookies and still not work at this time, it's weird...
Anyway, thank you for your time ;)

**Comment by sysroot255** [comment]
Hello everybody,

I just have try too your links ( @@||liens-telechargement.com^$elemhide) on TAB " My filters", apply, clean cache of ublock and updated after that i have also clean caches ands cookies from firefox , no issues, ublock was detected again and again, can you try to see from the beginning at this page and just try to download anything to see the real problem : https://ww1.zone-telechargement.ws/telecharger-series/31033-wynonna-earp-saison-2-french-webrip.html

redirect to : https://www.liens-telechargement.com/123455600123455602123455610123455615dj2otfkw1xds

 thanks in advance for any answers

**Comment by jspenguin2017** [comment]
Can someone post a screen recording? I don't understand what is the problem.

**Comment by okiehsch** [comment]
`https://www.liens-telechargement.com/123455601123455603123455610123455615123455617wg9zwk9wca`

![adb](https://user-images.githubusercontent.com/16838044/32868466-b5c57fe8-ca72-11e7-9bf3-1e2ef6618e34.JPG)


````
liens-telechargement.com##script:inject(abort-current-inline-script.js, jQuery, desactiver)
liens-telechargement.com##script:inject(abort-on-property-write.js, Fingerprint2)
liens-telechargement.com##script:inject(setTimeout-defuser.js, getElementBy)
liens-telechargement.com##script:inject(window.open-defuser.js)
liens-telechargement.com##.trtbl
liens-telechargement.com##a[href^="https://www.dl-protecte.com/"]
````

works for me.

**Comment by jspenguin2017** [comment]
Seems to be caught in generic defuser on my side. 
![image](https://user-images.githubusercontent.com/7283682/32868967-cd8519d6-ca31-11e7-9c0e-548a2fbaa61c.png)

**Comment by okiehsch** [comment]
What happens on your side if you add only
`liens-telechargement.com##a[href^="https://www.dl-protecte.com/"]` to deal with the "fake" stuff?

**Comment by uBlock-user** [comment]
![](https://i.gyazo.com/61c7bc017e235dd39992e17d0a5c1336.png)

Works fine on my side, run the logger and see if the filter is getting applied or not.

**Comment by uBlock-user** [comment]
@jspenguin2017 yeah the extension kills it via the generic detection, pinged you by mistake.

**Comment by okiehsch** [comment]
@uBlock-user he probably uses `EasyList Liste FR`, in which case your filter will not work because of 
`liens-telechargement.com##a[style]` in that list.

**Comment by uBlock-user** [comment]
Yes that and he's also on Firefox, so I can't confirm the same behaviour as my filters are tested on Chromium only.

**Comment by sysroot255** [comment]
I was to see on the forum “Easylist FR” it had indicated the following filter which functions perfectly

https://forums.lanik.us/viewtopic.php?f=91&t=39018

```

@@||ajax.googleapis.com^$script,domain=liens-telechargement.com
liens-telechargement.com#@#a[style]
```


a question : Are the lists updated automatically via ublock origin?

**Comment by jspenguin2017** [comment]
Yes, uBO will fetch list updates every 5 days unless otherwise instructed (by the metadata of the list).

**Comment by GLoBoLLo** [comment]
Hello,
I use Easylist FR.
I tried all solutions and only this one works :
```
liens-telechargement.com##script:inject(abort-current-inline-script.js, jQuery, desactiver)
liens-telechargement.com##script:inject(abort-on-property-write.js, Fingerprint2)
liens-telechargement.com##script:inject(setTimeout-defuser.js, getElementBy)
liens-telechargement.com##script:inject(window.open-defuser.js)
liens-telechargement.com##.trtbl
liens-telechargement.com##a[href^="https://www.dl-protecte.com/"]
```
THANK YOU! No more crappy stuff when I click, it's great ! 👍

**Comment by uBlock-user** [comment]
Just disable Easylist FR and my filter will work 

or

Disable the filter itself like this

```
liens-telechargement.com#@#a[style]
@@||liens-telechargement.com^$elemhide
```

**Comment by GLoBoLLo** [comment]
Yes thanks, this two solutions works fine. 
It's not the same result for me. Result of a test on the same link.
```
liens-telechargement.com##script:inject(abort-current-inline-script.js, jQuery, desactiver)
liens-telechargement.com##script:inject(abort-on-property-write.js, Fingerprint2)
liens-telechargement.com##script:inject(setTimeout-defuser.js, getElementBy)
liens-telechargement.com##script:inject(window.open-defuser.js)
liens-telechargement.com##.trtbl
liens-telechargement.com##a[href^="https://www.dl-protecte.com/"]
```
![capture1](https://user-images.githubusercontent.com/33693552/32946659-b6a0cbee-cb98-11e7-8d98-8cabed0019ee.jpg)

--------------
```
liens-telechargement.com#@#a[style]
@@||liens-telechargement.com^$elemhide
```
![capture2](https://user-images.githubusercontent.com/33693552/32946929-ce79b2ac-cb99-11e7-8d34-a5b036997283.jpg)

**Comment by GLoBoLLo** [comment]
Hello,
I prefer keep Easylist FR, cause I'm french ;)
```
liens-telechargement.com#@#a[style]
@@||liens-telechargement.com^$elemhide
```
Work fine on the first click, if i click a second time one the same or other link (it's usual for me when links are dead) the ad blocker is back.

Tested on this link :  https://ww1.zone-telechargement.ws/films-gratuit/31060-la-matresse-du-diable-french-hdrip.html

First click on the link Uptobox
![capture2_github com_reekanti-adblock-killer_issues_3743](https://user-images.githubusercontent.com/33693552/32947405-f89ede52-cb9b-11e7-8802-dbd4ddf0ad83.jpg)

I close tab (same if it not closed) and second click on the same link (same if I click another link)
![capture3](https://user-images.githubusercontent.com/33693552/32947413-040a8f0c-cb9c-11e7-8986-39236369d7bf.jpg)

With
```
liens-telechargement.com##script:inject(abort-current-inline-script.js, jQuery, desactiver)
liens-telechargement.com##script:inject(abort-on-property-write.js, Fingerprint2)
liens-telechargement.com##script:inject(setTimeout-defuser.js, getElementBy)
liens-telechargement.com##script:inject(window.open-defuser.js)
liens-telechargement.com##.trtbl
liens-telechargement.com##a[href^="https://www.dl-protecte.com/"]
```
It works all the time, and blocks all useless stuff
![capture1_github com_reekanti-adblock-killer_issues_3743](https://user-images.githubusercontent.com/33693552/32947899-82c9855e-cb9d-11e7-88c6-8b88db9e7c11.jpg)

By the way, a big thank at all, you are really reactive.

**Comment by GLoBoLLo** [comment]
I definitely have trouble with this button...

**Comment by sysroot255** [comment]
Rather test that instead of what you had put:
Essaye plutôt cela à la place de ce que tu avais mis :

@@||ajax.googleapis.com^$script,domain=liens-telechargement.com
 liens-telechargement.com#@#a[style]

**Comment by uBlock-user** [comment]
I'm on Chromium and that doesn't happen once I apply my filter, so it definitely has something to do with the Firefox version you're using. Anyways if those filters are working for you, then better them.

**Comment by GLoBoLLo** [comment]
Hello,
```
@@||ajax.googleapis.com^$script,domain=liens-telechargement.com
liens-telechargement.com#@#a[style]
```
Same result as :
_Même résultat que :_
```
liens-telechargement.com#@#a[style]
@@||liens-telechargement.com^$elemhide
```
I keep :
_Je garde :_
```
liens-telechargement.com##script:inject(abort-current-inline-script.js, jQuery, desactiver)
liens-telechargement.com##script:inject(abort-on-property-write.js, Fingerprint2)
liens-telechargement.com##script:inject(setTimeout-defuser.js, getElementBy)
liens-telechargement.com##script:inject(window.open-defuser.js)
liens-telechargement.com##.trtbl
liens-telechargement.com##a[href^="https://www.dl-protecte.com/"]
```
It work well and don't load useless stuff.
Thank you all 👍 
_Il fonctionne bien et ne charge pas les trucs qui ne servent à rien.
Merci à tous 👍_

**Comment by dreamingthink** [comment]
Hello,

I have tested and it is working with the 4 lines (and only these 4 lines) :
liens-telechargement.com#@#a[style]
@@||zone-telechargement.ws/uploads/dl.png
@@||liens-telechargement.com^$generichide
liens-telechargement.com##.spo-btn-link

**Comment by GLoBoLLo** [comment]
Yes, it works well on my system too. 👍 
However, I keep the one posted by **okiehsch**, the page loads a little faster, since it remains only the bare necessities.

_Oui, il fonctionne bien aussi sur mon système. 👍 
Cependant, je garde celui posté par **okiehsch**, la page se charge un peu plus vite, vu qu'il ne reste que le strict nécessaire._

**Comment by andykimpe** [comment]
thank you GLoBoLLo

I confirm working of this solution pull request send

https://github.com/reek/anti-adblock-killer/pull/3757

merci GLoBoLLo

je confirme le fonctionnement de cette solution pull request envoyer

https://github.com/reek/anti-adblock-killer/pull/3757
